rboot-sampleproject/main.c

```diff
@@ -18,9 +18,6 @@
 
 static os_timer_t network_timer;
 
-void ICACHE_FLASH_ATTR user_rf_pre_init() {

```

Comment:
D:\Projects\esp8266\rboot-sampleproject>make
CC main.c
CC uart.c
CC rboot-ota.c
LD rom0.elf
C:/esp_iot_sdk_v1.2.0/lib\libmain.a(app_main.o): In function `user_uart_wait_tx_fifo_empty':
(.irom0.text+0x324): undefined reference to `user_rf_pre_init'
C:/esp_iot_sdk_v1.2.0/lib\libmain.a(app_main.o): In function `user_uart_wait_tx_fifo_empty':
(.irom0.text+0x443): undefined reference to `user_rf_pre_init'
collect2.exe: error: ld returned 1 exit status
Makefile:45: recipe for target 'build/rom0.elf' failed
make: *** [build/rom0.elf] Error 1